raiden/connection_manager.py

```diff
@@ -223,10 +232,9 @@ def retry_connect(self):
         If the connection manager has no funds, this is a noop.
         """
         with self.lock:
-            if self._funds_remaining <= 0 or self._leaving_state:
-                return
-
-            self._open_channels()
+            while self._funds_remaining > 0 and not self._leaving_state:
+                if self._open_channels() is False:
```

Comment:
Being nitpicky, I think there's no need to be so restrict with `is`, a `not self._open_channels()` would be enough

Reply:
ok, I changed it because I had to repush due to CLA not reporting its status. 

To me the code is more readable (visually) if the method returns bool and a boolean is explicitly stated in the condition.. But I get your point. ;)